### ISSUE TITLE
[CS 1.6] crashes when I use sv_cheats 1 on a local server 

### ISSUE THREAD

**Issue by TheRumble** [ValveSoftware/halflife#2853]
Title basically, the only thing I can think of is that I have a bunch of other goldsrc games installed and they all share the same problem 
Error is as follows:
SZ_GetSpace: Tried to write an uninitialized sizebuf_t: ???

**Comment by kisak-valve** [comment]
Possible duplicate of #2785.

**Comment by TheRumble** [comment]
> Possible duplicate of #2785.

Thanks, but I have no idea how to understand that, is it a solution to the error? If so, how do I apply it?

**Comment by hogsy** [comment]
> Thanks, but I have no idea how to understand that, is it a solution to the error? If so, how do I apply it?

No, it means the ticket you've raised looks to be a duplicate of another ticket that's been raised. Read.

**Comment by TheRumble** [comment]
> Thanks, but I have no idea how to understand that, is it a solution to the error? If so, how do I apply it?
> 
> No, it means the ticket you've raised looks to be a duplicate of another ticket that's been raised. Read.

I understand that my ticket is a near duplicate of the linked ticket. Sorry, I'm completely new to github and the format is strange. My question was how do I apply the other ticket, and whether it was a solution or not.

**Comment by hogsy** [comment]
> My question was how do I apply the other ticket, and whether it was a solution or not.

The solution specified in the ticket that was linked is one for the developers to implement, but there's nothing you can do. You'll need to wait for Valve to implement the fix.

If you could confirm whether or not it's a duplicate though would be very helpful. For example, are the steps used to reproduce the fault in the ticket that was linked the same steps you think you've used to reproduce the fault you encountered?

One step they've mentioned for example is this.

> You must have at least one fakeclient (zbot or something else) in your listenserver.

Would you say this was the case for you as well?

**Comment by deprale** [comment]
You have messed with client.dll Possibly gotten an older version or a version from another cs  series games( cscz for example)
